gno/p/role_manager/role_manager.gno

```diff
@@ -191,19 +163,10 @@ func (rm *RoleManager) mustGetRole(roleName string) *Role {
 	return role.(*Role)
 }
 
-func (rm *RoleManager) initUser(user std.Address) *avl.Tree {
-	uRole := rm.mustGetRole(RoleUser)
-	tree := avl.NewTree()
-	tree.Set(RoleUser, uRole)
-	rm.users.Set(user.String(), tree)
-	uRole.users.Set(user.String(), struct{}{})
-	return tree
-}
-
-func (rm *RoleManager) getUser(addr std.Address) *avl.Tree {
+func (rm *RoleManager) mustGetUser(addr std.Address) *avl.Tree {
 	user, ok := rm.users.Get(addr.String())
 	if !ok {
-		return nil
+		panic("user does not have any roles")
```

Comment:
I think we should not panic here, so we can just return false in `HasRole` and `HasPermission` if `userRoles == nil`
they will be hard to use otherwise I think

`RemoveRoleFromUser` could be noop too if user doesn't exists instead of panic